src/app/cluster/duck/sagas.ts

```diff
@@ -561,7 +565,11 @@ function* initDiscoveryCert() {
       yield delay(CLUSTER_RETRY_PERIOD);
       continue;
     }
-    const discoveryClient: IDiscoveryClient = ClientFactory.discovery(state);
+    const discoveryClient: IDiscoveryClient = DiscoveryFactory.discovery(
+      state.auth.user,
+      state.auth.migMeta.namespace,
```

Comment:
Not sure if it's related to this change, but I got the following warning when loading the app.

<img width="552" alt="Screen Shot 2021-07-19 at 9 26 23 AM" src="https://user-images.githubusercontent.com/5942899/126167707-bb061c73-5b6d-4f7b-b991-f3c84517e27a.png">

<img width="866" alt="Screen Shot 2021-07-19 at 9 26 10 AM" src="https://user-images.githubusercontent.com/5942899/126167704-9a060f44-4f46-4978-b300-058b28bbf48e.png">

Reply:
Ah, looks like I was missing a config for `discoveryApi` - updated and everything is working now!